build/build-maven-publish.xml

```diff
@@ -179,9 +203,9 @@ SPDX-License-Identifier: Apache-2.0
         <!-- prepare the pom-file -->
         <generate-solver-pom-file/>
         <!-- then publish the files -->
-        <stage-solver-file filename="libboolector" fileending="so"/>
-        <stage-solver-file filename="libminisat" fileending="so"/>
-        <stage-solver-file filename="libpicosat" fileending="so"/>
+        <stage-solver-file filename="libboolector" classifier="libboolector" fileending="so"/>
```

Comment:
Maybe we should add the `x64` suffix for all existing solvers as well, so that we can discern their architecture more easily, and prepare adding ARM for more solvers. (Although we will most likely not support/get support for older solvers to be fair)

Reply:
I though about this topic as well and would do the following:
- If there is a (sufficiently) new version of a solver, where we could provide multiple architectures or need to compile the new  library anyway, then update the Ivy and Maven configurations and scripts.
- Otherwise, keep current state.

I did this with OpenSMT in the last few commits in this PR. It was quite easy to compile it for ARM64 after refactoring the script and playing around with the GMP dependency. :smile:

Reply:
I agree with your assessment!
It's nice that apparently multiple solvers work with ARM64 rather easily.